README.rst_bkp

```diff
@@ -0,0 +1,368 @@
+        with Cycamore. It relies on the input files that are part of your Cycamore
+        repository, and only succeeds for input files that are correct (some may have
+        known issues. See the issue list in cycamore for details.) To run the example
+        input files, ::
+
+          python ../install/cycamore/bin/run_inputs.py
+
+  * **Making a Pull Request**
+
+      - When you are ready to move changes from one of your topic branches into the
+        "develop" branch, it must be reviewed and accepted by another
+        developer.
+
+      - You may want to review this `tutorial <https://help.github.com/articles/using-pull-requests/>`_
+        before you make a pull request to the develop branch.
+
+  * **Reviewing a Pull Request**
+
+     - Build, install, and test it. If you have added the remmote repository as
+       a remote you can check it out and merge it with the current develop
+       branch thusly, ::
+
+         git checkout -b remote_name/branch_name
+         git merge develop
+
+     - Look over the code.
+
+        - Check that it meets `our style guidelines <http://cyclus.github.com/devdoc/style_guide.html>`_.
+
+        - Make inline review comments concerning improvements.
+
+     - Accept the Pull Request
+
+        - In general, **every commit** (notice this is not 'every push') to the
+          "develop" and "master" branches should compile and pass tests. This
+          is guaranteed by using a NON-fast-forward merge during the pull request
+          acceptance process.
+
+        - The green "Merge Pull Request" button does a non-fast-forward merge by
+          default. However, if that button is unavailable, you've made minor
+          local changes to the pulled branch, or you just want to do it from the
+          command line, make sure your merge is a non-fast-forward merge. For example::
+
+            git checkout develop
+            git merge --no-ff remote_name/branch_name -m "A message""
+
+Cautions
+========
+
+  * **NEVER** merge the "master" branch into the "develop"
+    branch. Changes should only flow *to* the "master" branch *from* the
+    "develop" branch.
+
+  * **DO NOT** rebase any commits that have been pulled/pushed anywhere
+    else other than your own fork (especially if those commits have been
+    integrated into the upstream repository.  You should NEVER rebase
+    commits that are a part of the 'master' branch.  *If you do, you will be
+    flogged publicly*.
+
+  * Make sure that you are pushing/pulling from/to the right branches.
+    When in doubt, use the following syntax::
+
+      git push [remote] [from-branch]:[to-branch]
+
+    and (*note that pull always merges into the current checked out branch*)::
+
+      git pull [remote] [from-branch]
+
+An Example
+==========
+
+Introduction
+------------
+
+As this type of workflow can be complicated to converts from SVN and very complicated
+for brand new programmers, an example is provided.
+
```

Comment:
. . . can be complicated to brand new programmers or those accustomed to SVN, an example . . . .